src/resolvers/UserResolver.js

```diff
@@ -0,0 +1,24 @@
+const resolvers = {
+  // Query: {
```

Comment:
Yeah, you can stick these in a note somewhere, I'd rather not have them in the `dev` branch, thanks @ronanfitz!